### PR TITLE
Translation upload

### PR DESCRIPTION
**Pull request by hellowenqi**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [x] Rebase before creating a PR to keep commit history clear.
- [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by benjycui** [comment]
需要 rebase...

File: components/upload/demo/basic.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 0
-title: 点击上传
+title: 
+  zh-CN: 点击上传
+  en-US: upload by clicking
```
Comment (benjycui) [inline components/upload/demo/basic.md]: Upload

注意大小写。。

File: components/upload/index.zh-CN.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 category: Components
 chinese: 上传
 type: Form Controls
-english: Upload
+english: Upload8/2/2016 11:05:18 AM
```
Comment (benjycui) [inline components/upload/index.zh-CN.md]: ？？？？？？

File: components/upload/index.zh-CN.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ english: Upload
 
 | 参数       | 说明                                                         | 类型        | 默认值|
 |------------|--------------------------------------------------------------| ----------- |-------|
-| name       | 可选参数, 上传的文件                                         | String      | file  |
+ name       | 可选参数, 上传的文件                                         | String      | file  |
```
Comment (benjycui) [inline components/upload/index.zh-CN.md]: ？？？？ markdown 格式有问题。

**Comment by benjycui** [comment]
rebase?

**Comment by benjycui** [comment]
Rebase and `push --force`...

你这样把别人的 commit 也带进来了，不好 review...

**Comment by benjycui** [comment]
Close, trace https://github.com/ant-design/ant-design/pull/2770

**Comment by afc163** [comment]
@hellowenqi 你[不关联 email](https://github.com/ant-design/ant-design/pull/2707#issuecomment-240729690) 的话，会无法出现在贡献者名单里的：https://github.com/ant-design/ant-design/graphs/contributors

**Pull request closed without merging**